lib/linguist/languages.yml

```diff
@@ -2736,6 +2736,20 @@ Go Module:
   tm_scope: go.mod
   ace_mode: text
   language_id: 947461016
+Go Template:
+  type: markup
+  color: "#00ADD8"
+  extensions:
+  - ".gohtml"
```

Comment:
See the comment at the top of this file. The first extension should be the language's primary extension. Your comments and the fact you've not provided a sample or supporting search result suggests this is not the primary extension for the language.